src/WebJobs.Extensions.DurableTask/HttpCreationPayload.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// Data structure containing orchestration instance creation HTTP endpoints.
+    /// </summary>
+    public class HttpCreationPayload
+    {
+        /// <summary>
+        /// Gets the HTTP POST orchestration instance creation endpoint URL.
+        /// </summary>
+        /// <value>
+        /// The HTTP URL for creating a new orchestration instance.
+        /// </value>
+        [JsonProperty("createNewInstancePostUri")]
+        public string CreateNewInstancePostUri { get; internal set; }
+
+        /// <summary>
+        /// Gets the HTTP POST orchestration instance create-and-wait endpoint URL.
+        /// </summary>
+        /// <value>
+        /// The HTTP URL for creating a new orchestration instance and waiting on its completion.
+        /// </value>
+        [JsonProperty("waitOnNewInstancePostUri")]
+        public string WaitOnNewInstancePostUri { get; internal set; }
```

Comment:
I had to scratch my head for a minute on this one. Can we name it `CreateAndWait...` so that the name better matches the behavior?